services/QuillCMS/app/workers/rematch_response_worker.rb

```diff
@@ -16,6 +19,8 @@ class RematchResponseWorker
     'spelling_error' => false
   }.freeze
 
+  class LambdaError < StandardError; end
```

Comment:
How about `Not200Error`? `Lambda` is a bit cryptic.

Reply:
Good point. I renamed this to `LamdaHTTPError` since that is more clear (and better matches the `Net::HTTPError` it replaces).